configs/projects/_shared-compiler-settings.rb

```diff
@@ -12,7 +12,6 @@
 # are too old to support these flags.
 
 if((platform.is_sles? && platform.os_version.to_i >= 15) ||
-      (platform.is_el? && platform.os_version.to_i == 8 && platform.architecture !~ /ppc64/) ||
```

Comment:
Does this mean RHEL 9 ppc now supports pie? If so, the commit message should mention the change in behavior.